File: VB/WindowsApplication3/Common/ModuleDescription.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace DXSample
    Public Class ModuleDescription
        Public Sub New(ByVal documentType As String, ByVal title As String)
            Me.DocumentType = documentType
            Me.Title = title
        End Sub
        Public Property DocumentType() As String
        Public Property Title() As String
    End Class
End Namespace

File: CS/WindowsApplication3/Common/ModuleDescription.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DXSample {
    public class ModuleDescription {
        public ModuleDescription(string documentType, string title){
            DocumentType = documentType;
            Title = title;
        }
        public string DocumentType { get; set;}
        public string Title { get; set; }
    }
}
